org.eclipse.m2e.core.ui/META-INF/MANIFEST.MF

```diff
@@ -6,17 +6,23 @@ Bundle-RequiredExecutionEnvironment: JavaSE-17
 Bundle-Name: %Bundle-Name
 Bundle-Vendor: %Bundle-Vendor
 Bundle-Localization: plugin
-Export-Package: org.eclipse.m2e.core.ui.internal;x-friends:="org.eclipse.m2e.editor",
+Export-Package: org.eclipse.m2e.core.ui.internal;x-friends:="org.eclipse.m2e.editor,org.eclipse.m2e.scm",
  org.eclipse.m2e.core.ui.internal.actions;
   x-friends:="org.eclipse.m2e.editor,
    org.eclipse.m2e.editor.xml,
    org.eclipse.m2e.scm,
-   org.eclipse.m2e.jdt.ui",
+   org.eclipse.m2e.jdt.ui,
+   org.eclipse.m2e.tests,
+   org.eclipse.m2e.editor.tests",
```

Comment:
For me it is a bit strange, to mark 'test-plugins' as x-friends. Can't you instead disbale discouraged access-warnings in the preferences of the tests, just like you did in the apt.tests?

Applies to other 'production' plugins too.